pills/00-preface.md

```diff
@@ -1,10 +1,13 @@
 # Preface
 
-This is a ported version of the **Nix Pills**, a series of blog posts written by **Luca Bruno** (aka Lethalman) and originally published in 2014 and 2015. It provides a tutorial introduction into the Nix package manager and Nixpkgs package collection, in the form of short chapters called 'pills'.
+This is a ported version of the **Nix Pills**, a series of blog posts written by **Luca Bruno** (aka Lethalman) and originally published in 2014 and 2015.
+It provides a tutorial introduction into the Nix package manager and Nixpkgs package collection, in the form of short chapters called 'pills'.
 
-Since the Nix Pills are considered a classic introduction to Nix, an effort to port them to the current format was led by Graham Christensen (aka grahamc / gchristensen) and other contributors in 2017.
+Since the Nix Pills are considered a classic introduction to Nix, an effort to port them to a DocBook format was led by Graham Christensen (aka grahamc / gchristensen) and other contributors in 2017.
```

Comment:
Why 2 pseudos for Graham ?

Reply:
I don't know, I'm just keeping the information that was there before.

Reply:
@grahamc Should we keep the 2 pseudonames or one is enough here?

Reply:
grahamc is great, thanks! :)